docs/flink/flink-getting-started.md

```diff
@@ -189,11 +190,11 @@ table_env.execute_sql("CREATE CATALOG my_catalog WITH ("
                       "'my-additional-catalog-config'='my-value')")
 ```
 
-For more details, please refer to the [Python Table API](https://ci.apache.org/projects/flink/flink-docs-release-1.13/docs/dev/python/table/intro_to_table_api/).
+For more details, please refer to the [Python Table API](https://ci.apache.org/projects/flink/flink-docs-release-1.14/docs/dev/python/table/intro_to_table_api/).
 
 ## Creating catalogs and using catalogs.
 
-Flink 1.11 support to create catalogs by using flink sql.
+Flink 1.14 support to create catalogs by using flink sql.
```

Comment:
Maybe the version number can be dropped here? I think at one point in time Flink didn't support creating catalogs via SQL, only the YAML files.

So this is simply one less thing to keep updating if we remove the version number from it.

Reply:
@kbendick Thank you for your review, now I have revised all the questions, please check thank you.

Reply:
@kbendick  I want to upgrade to flink 1.15.0, but I find some resources are not available yet, so I still use 1.14.4.

Reply:
@kbendick Do i have something not modify ? please tell me : )